src/views/AddItem.jsx

```diff
@@ -1,3 +1,65 @@
+import { useState } from 'react';
+import { addItem } from '../api/firebase.js';
+
 export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+	const [itemName, setItemName] = useState('');
+	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(7);
+	const [itemAdded, setItemAdded] = useState(false);
+	const [error, setError] = useState(false);
+
+	const handleSubmit = async (e) => {
```

Comment:
Nice job setting up this `handleSubmit` function! One thing to consider for the future - we'll need to dynamically pass in the list token to make sure items are added to the right list. We can abstract that away here, or maybe add a `TODO` comment about needing to do it in the future.